pkg/controllers/manageagent/manageagent_test.go

```diff
@@ -0,0 +1,276 @@
+package manageagent
+
+import (
+	"testing"
+
+	// counterfeiter -o internal/fakes github.com/rancher/wrangler/pkg/generated/controllers/core/v1.NamespaceController
```

Comment:
can we remove the counterfeiter comment as we are using gomock?

Reply:
I turned that into a `//go:generate` comment. `go generate ./pkg/controllers/manageagent` will re-create the mock.